src/unicef_vision/loaders.py

```diff
@@ -32,7 +30,10 @@ def set_url(self, endpoint, detail):
         logger.info('About to get data from {}'.format(self.url))
 
     def set_headers(self, headers):
-        self.headers = {'Content-Type': 'application/json'}
+        self.headers = {
```

Comment:
Any reason not to have this header in a single spot and reuse?

Reply:
what do you mean?

Reply:
```
{
    'Content-Type': 'application/json',
    'Ocp - Apim - Subscription - Key': settings.INSIGHT_SUB_KEY
}
```
Is used on the following lines;
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/src/unicef_vision/utils.py#L24
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L26
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L84
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L130